app/views/periodic_expenses/index.slim

```diff
@@ -5,15 +5,36 @@
 - else
 
   table
```

Comment:
It's almost the same, it has a few different fields. For instance, PeriodicExpense will always have a period while Expense will never have it.